test/tests.py

```diff
@@ -305,3 +310,32 @@ def test_0006_OSB(self):
             if source_count != target_count:
                 self.assertEqual(source_count, target_count, f'{index}: doc counts do not match - '
                                                              f'Source = {source_count}, Target = {target_count}')
+
+    def test_0007_timeBetweenRequestsOnSameConnection(self):
+        # This test will verify that the replayer functions correctly when
+        # requests on the same connection on the proxy that has a minute gap
+        seconds_between_requests = 60  # 1 minute
```

Comment:
This test will take at least 180 seconds to run?  Is there anyway the system can be setup with shorter limits such as 10 second buffers that would still validate the scenario, but only take ~30 to run?

Reply:
This test specifically is waiting a minute between requests as a somewhat arbitrary value. For the purposes of the included bug, i was interested in the delay being > than target connection timeout, which is set at the default of 30 seconds.

As is, i could reduce this delay to 35 seconds and achieve that goal, or we could set up infrastructure to reduce that and make a constrained test.

My thoughts is that the shorter limits tests would be better as a unit test possibly with docker containers, whereas this is intended to validate broad scenarios which may cover different bugs

Reply:
How do we know 60 second delay is correct to evaluate vs a 90 second delay would actually show an issue?

Reply:
We don't it's arbitrary

Reply:
So why are we writing this test, did we have an issue in the past this would have caught?  Could we revert the fix and confirm the test would validate that scenario?

Reply:
We did not have a specific behavior bug that this would have caught, but there was a component which was broken and this test triggered the error case showing a "ReadTimeoutException" in the log. That I did tests (as i created this test case prior to pushing out that other fix)

Reply:
I guess my concern would be - if that scenario happened again do we have enough information to diagnose what took place, but at this point it feels like I am being pedantic, I'd rather have the test running to prompt investigation and improvement to data collection troubleshooting.